packages/hackmcx-api/src/db/migrations/2022-03-08_1129_create_table_captions.cjs

```diff
@@ -0,0 +1,21 @@
+function up (knex) {
+    return knex.schema
+        .createTable('captions', function (table){
+            table.increments('id');
+            table.string('caption', 2048).notNullable();
+            table.integer('post_id').unsigned();
+            table.foreign('post_id').references('id').inTable('posts').onDelete('cascade');
+            table.float('average_rating',2,1);
```

Comment:
We just want to grab the defaults here, it should provide less compatibility issues. 

```suggestion
            table.float('acc_rating');
```